### ISSUE TITLE
MessageNotUnderstood exception on launch

### ISSUE THREAD

**Issue by avasopht** [pharo-project/pharo-launcher#119]
From a fresh install of Pharo Launcher on Windows 10 Pro to `c:\pharo-launcher` (or any other directory) with antivirus disabled I get a `MessageNotUnderstood: receiver of "launcherStartUp" is nil` exception, requiring me to make the following change for launcher to start:

```Smalltalk
"PhLStartupManager class"
startUp
    self startUpClasses do: [ :c | c ifNotNil: [ c launcherStartUp ] ]
```

Pharo Launcher had been working fine until just a few days ago. Now this bug persists no matter where I install Pharo Launcher. I've even tried deleting the Pharo folder in Documents, but I still get this issue.

What I did notice was that `self startUpClasses` had something like 6 / 10 `nil` references!

**Comment by VincentBlondeau** [comment]
What does AV means for you?

**Comment by avasopht** [comment]
Antivirus is disabled.

**Comment by demarey** [comment]
Thanks for the report and the fix.
It will integrated in next release.

--------------
Name: PharoLauncher-Core-ChristopheDemarey.197
Author: ChristopheDemarey
Time: 19 June 2018, 11:22:07.771079 am
UUID: 868efef3-5c2b-0d00-a04a-aa780bca7595
Ancestors: PharoLauncher-Core-CyrilFerlicot.196

fix#119  MessageNotUnderstood exception on launch.
Contributed by Keldon Alleyne, thanks!
